make.xsh

```diff
@@ -133,30 +133,55 @@ def start_backend_prod():
     __start_backend(env_file=PROD_ENV_FILE)
 
 @makefile.target(tag="run services locally")
-def backup_database_dev():
+def run_local_db():
     """
     Use the "database-backup" service in the "docker-compose.yml" file to backup
     all of the tables in the instace of Postgres running locally.
     """
     export_dot_env_vars(env_file=DEV_ENV_FILE)
+    export_rootski_profile_aws_creds_db_backup()
     $POSTGRES_HOST = get_localhost()
-    docker network prune --force
+    docker-compose run -d -p $POSTGRES_PORT:$POSTGRES_PORT postgres
+
+@makefile.target(tag="run services locally")
+def backup_local_db():
+    """
+    Use the "database-backup" service in the "docker-compose.yml" file to backup
+    all of the tables in the instace of Postgres running locally. You must have a
+    postgres container running which you can run using "make run-local-db"
+    """
+    export_dot_env_vars(env_file=DEV_ENV_FILE)
+    export_rootski_profile_aws_creds_db_backup()
+    $POSTGRES_HOST = get_localhost()
+    # Runs a postgres instance if one is not already running
+    docker ps | grep postgres || docker-compose run -d -p $POSTGRES_PORT:$POSTGRES_PORT postgres
     docker-compose run database-backup backup
 
 @makefile.target(tag="run services locally")
-def restore_database():
+def restore_local_db():
+    """
+    Use the "database-backup" service in the "docker-compose.yml" file to backup
+    all of the tables in the instace of Postgres running locally.
     """
-    Use the "database-backup" service in the "docker-compose.yml" file to identify
-    the most recent backup in "infrastructure/containers/postgres/backups" and load that data
-    into the database.
+    export_dot_env_vars(env_file=DEV_ENV_FILE)
+    export_rootski_profile_aws_creds_db_backup()
+    $POSTGRES_HOST = get_localhost()
+    # Runs a postgres instance if one is not already running
+    docker ps | grep postgres || docker-compose run -d -p $POSTGRES_PORT:$POSTGRES_PORT postgres
+    docker-compose run database-backup restore-from-most-recent
 
-    Note that this will wipe the existing data in the database first.
+@makefile.target(tag="run services locally")
+def backup_local_db_on_interval():
```

Comment:
Technically the database is "local" whether it's running for dev or prod. If we just had a target called `make start-database-stack` that started up both the database and the database backup process, I think that would achieve the same thing as both of these targets and we could use it for both local testing and prod which would be really handy.

Reply:
It would need to use `docker stack deploy ...` in that case instead of `docker-compose up` or `docker-compose run`